src/alchemlyb/parsing/amber.py

```diff
@@ -252,63 +248,94 @@ def extract_u_nk(outfile, T):
         Temperature in Kelvin at which the simulations were performed;
         needed to generated the reduced potential (in units of kT)
 
-    Returns
+    Returns a dictionary with elements:
```

Comment:
Please don't change this as this would break sphinx.
Just say return a dictionary. Might also be good to check if the doc builds correctly.

```
Returns
-------
Dict
    A dictionary with keys of 'u_nk', which is a pandas DataFrame ....
```

Reply:
I'm sorry, I didn't know about that :-)

Reply:
I edited the docstring for every new `extract` functions, and added to the docs, now the `extract` function appears correctly when the docs are built.